lib/pom.xml

```diff
@@ -19,15 +19,15 @@
         <version>3.0.2</version>
         <configuration>
           <outputDirectory>${project.build.outputDirectory}/resources</outputDirectory>
-      </configuration>
+        </configuration>
```

Comment:
Since we change in this PR the version to `0.0.4` in CHANGELOG.md, let's update the version in the `pom.xml` file to be consistent.

Reply:
Yes, I will do that after we merge. We can publish a new version.